.github/workflows/deploy-release.yaml

```diff
@@ -0,0 +1,32 @@
+name: deployment
```

Comment:
For a better differentiation, we should rename the tasks differently - right now both of them appear as "deployment" in the Github Actions Log:

maybe by removing the root level name it just falls back to the yaml name, whic hwould be totally appropriate here.